Platforms/QemuQ35Pkg/.azurepipelines/NuGet.yml

```diff
@@ -0,0 +1,204 @@
+## @file
+# Azure Pipeline build file for publishing a NuGet package.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+- name: publish_q35_binaries
+  displayName: Publish QEMU Q35 Binaries
+  type: boolean
+  default: true
+- name: publish_q35sbsa_binaries
+  displayName: Publish QEMU Q35 SBSA Binaries

```

Comment:
I don't think we've been using Q35 in the naming for SBSA since thats specific to the x64 machine type.

Reply:
That makes sense. Wasn't really thinking about it while doing this. I'll update it.

Reply:
Updated in PR commit https://github.com/microsoft/mu_tiano_platforms/pull/286/commits/3ba367854e17ccff03aa14b948fc471232c5031d